.github/workflows/shaptools-ci.yml

```diff
@@ -0,0 +1,103 @@
+name: Formula CI
+# - this workflow will
+# - test python version matrix [2.7,3.6, 3.7, 3.8, 3.9] for coverage 
+# - uploads coverage data to codeClimate --> Results are linked  as a  badges in the README
+# - deliver the package content to the configured repository
+# - submit the new package content to the upstream repository
+on: [push, pull_request]
+env:
+  PACKAGE_NAME: python-shaptools
+  TAR_NAME: shaptools
+jobs:
+  
+  testPython:
```

Comment:
I don't personally like camecase here. I would prefer `test-python` for instance

Reply:
Oki I will update it, the job name is okay ? Or should be they called more specific ?